### PR TITLE
gpg4win: fix checkver

### PR DESCRIPTION
**Pull request by linsui**
gpg4win need privilege.

```
ERROR Exception calling "Start" with "0" argument(s): "The requested operation requires elevation" 
Installation aborted. You might need to run 'scoop uninstall gpg4win' before trying again.
```

### CODE REVIEW

File: bucket/gpg4win.json [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
-    "homepage": "https://www.gpg4win.org",
-    "description": "GNU Privacy Guard for Windows is encryption software for files and emails.",
+    "homepage": "https://gpg4win.org",
+    "description": "Encryption software for files and emails.",
     "license": "GPL-2.0-or-later",
-    "url": "https://files.gpg4win.org/gpg4win-3.1.10.exe",
+    "url": "https://files.gpg4win.org/gpg4win-3.1.10.exe#/gpg4win.exe",
     "hash": "7f3ceaf54bcff0f63716900f6769b93597aae33e188e88792aa484a93d056dc9",
+    "installer": {
+        "script": "Invoke-ExternalCommand -FilePath \"$dir\\gpg4win.exe\" -ArgumentList \"/S /D=$dir\\Gpg4win\" -RunAs | Out-Null"
```
Comment (Ash258) [inline bucket/gpg4win.json]: Use $fname

File: bucket/gpg4win.json [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
     "version": "3.1.10",
-    "homepage": "https://www.gpg4win.org",
-    "description": "GNU Privacy Guard for Windows is encryption software for files and emails.",
+    "homepage": "https://gpg4win.org",
+    "description": "Encryption software for files and emails.",
     "license": "GPL-2.0-or-later",
-    "url": "https://files.gpg4win.org/gpg4win-3.1.10.exe",
+    "url": "https://files.gpg4win.org/gpg4win-3.1.10.exe#/gpg4win.exe",
     "hash": "7f3ceaf54bcff0f63716900f6769b93597aae33e188e88792aa484a93d056dc9",
+    "installer": {
```
Comment (Ash258) [inline bucket/gpg4win.json]: Add pre_install block with check for admin and information about admin access

**Comment by linsui** [comment]
The checkver url is changed. 

https://scoop.r15.ch/extras/mud-20191110-080001.log

```
gpg4win: No such device or address No such device or address
URL https://www.gpg4win.org/get-gpg4win.html is not valid
```

It seems that https://www.gpg4win.org is broken but has been fixed now. I can access it.

**Comment by Ash258** [comment]
And second following question. Why do you fix 10 different things in 1 PR? This could easily be 2 PRs with appropriate Title and with clean history.

TItle say checkver fix, and suddently there is some installer changes with -RunAs which is against pure basic scoop state of its homepage.

**Comment by linsui** [comment]
Ok, I'll just keep the first commit. So how to deal with gpg4win and other app which need privilege to install? Without -RunAs installation will failed.

**Comment by Ash258** [comment]
https://github.com/lukesampson/scoop-extras/pull/3144#discussion_r344478397

**Comment by linsui** [comment]
www.gpg4win.org works now. Close this.

**Pull request closed without merging**